client/src/components/files/FileCard.vue

```diff
@@ -1,12 +1,23 @@
 <!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
 
 <template>
-  <div
-    class="card"
+  <ion-item
+    class="card-file-item ion-no-padding"
+    :class="{ selected: isSelected }"
```

Comment:
The select for `FileCard` seems to be displayed correctly, and the action buttons change depending on the selection, but none of them work.

Reply:
After investigating, a props was missing, but now it works.